src/models/tasks.py

```diff
@@ -0,0 +1,37 @@
+from transformers import BertConfig, BertForSequenceClassification
+import torch
+from torch import nn
+from loguru import logger
+
+from src.models.vtr.ocr import OCRHead
+from src.utils.common import PositionalEncoding
+
+
+class SequenceClassifier(nn.Module):
+    def __init__(self, config, embedder, ocr: OCRHead = None):
+        super().__init__()
+
+        logger.info(
+            f"Initializing vanilla BERT classifier | hidden size: {config['hidden_size']}, "
+            f"# layers: {config['num_hidden_layers']}"
+        )
+
+        model_config = BertConfig(**config)
+        self.backbone = BertForSequenceClassification(model_config)
+        self.embedder = embedder
+        self.ocr = ocr
```

Comment:
Please, add here an additional check: if `ocr` is passed then the embedder should be an instance of `VTREmbedder`